source/slang/hlsl.meta.slang

```diff
@@ -12305,6 +12359,14 @@ uint4 WaveMatch(T value)
     __target_switch
     {
     case hlsl: __intrinsic_asm "WaveMatch";
+    case glsl: __intrinsic_asm "subgroupPartitionNV($0)";
```

Comment:
is subgroupPartitionNV part of HLSL as well?

Reply:
This is implementing the intrinsic for the `case glsl` case, so it is for GLSL.

Reply:
oh i thought HLSL was pass through case.

Reply:
__intrinsic_asm is like a `return`, and there is no passthrough for __target_switch.